tests/ts_B_select_pass_3_sorted_on_key.erl

```diff
@@ -6,22 +6,25 @@
 
 -behavior(riak_test).
 
+-include_lib("eunit/include/eunit.hrl").
+
 -export([confirm/0]).
 
 confirm() ->
-    Cluster = multiple,
+    ClusterType = multiple,
     TestType = normal,
-    DDL = timeseries_util:get_ddl(docs),
-    Data = timeseries_util:get_valid_select_data(),
+    DDL = ts_util:get_ddl(docs),
+    Data = ts_util:get_valid_select_data(),
     ShuffledData = shuffle_list(Data),
-    Qry = timeseries_util:get_valid_qry(),
+    Qry = ts_util:get_valid_qry(),
     Expected = {
-        timeseries_util:get_cols(docs),
-        timeseries_util:exclusive_result_from_data(Data, 1, 9000000000)},
+        ts_util:get_cols(docs),
+        ts_util:exclusive_result_from_data(Data, 2, 9)},
```

Comment:
Did you shrink this range for expediency while testing the test? Should it be expanded again?